plugins/ps/functions

```diff
@@ -20,18 +33,16 @@ generate_scale_file() {
     TMP_WORK_DIR=$(mktemp -d)
     trap 'rm -rf "$TMP_WORK_DIR" > /dev/null' RETURN
 
-    copy_from_image "$IMAGE" "/app/Procfile" "$TMP_WORK_DIR" 2>/dev/null || true
-    local PROCFILE="$TMP_WORK_DIR/Procfile"
-    if [[ ! -e "$PROCFILE" ]]; then
-      echo "web=1" >> $DOKKU_SCALE_FILE
-    else
+    if [[ -f $DOKKU_PROCFILE ]]; then
       while read -r line || [[ -n "$line" ]]; do
-        [[ "$line" =~ ^#.* ]] && continue
+        if [[ -z "$line" ]] || [[ "$line" == "#"* ]]; then continue; fi
```

Comment:
Sure, the old way wasn't skipping empty lines. This also is what was in that other bask procfile parser (shoreman). I'm happy to make it a well indented if block though :)